src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -52,6 +54,15 @@ static XunitBenchmark()
                     ),
                 }
             };
+
+            foreach (var pmc in PerformanceMonitorCounters)
```

Comment:
Yup, no issue with forcing it on.  Just wanted to make sure that PMC sources can still be enabled via the existing mechanism.